docs/user/data_management.rst

```diff
@@ -160,6 +160,37 @@ If you are using the GeoMesa ``SftBuilder``, you may call the overloaded attribu
         .geometry("geom", default = true)
         .build("mySft")
 
+.. _configuring_z_shards:
+
+Configuring Z-Index Shards
+--------------------------
+
+GeoMesa allows you to specify the number of shards (or splits) to be used for each SimpleFeatureType. This gives
+you more flexibility with SimpleFeatureTypes whether they are large or small. By default, SimpleFeatureTypes are
+created with 4 as the number of shards. 4 is the default because previous versions of GeoMesa hardcoded it in
+the z indices.
```

Comment:
How about this:

"GeoMesa Accumulo permits one to specify the number of shards (or splits) into which the Z2 (or Z3) indices are divided. This parameter may be changed individually for each ``SimpleFeatureType``. In previous versions of GeoMesa, the number of shards was "hard-coded" set to 4, which remains the default. Valid values for the number of shards range from 1 to 127 inclusive.

When using very large data sets or large clusters, it may be of benefit to use a larger number of shards. However, in most test cases--especially with single-node or small clusters--the default of 4 should be preferred.

The number of shards is set..." and so on.